### PR TITLE
DOCS-4338

### PR DESCRIPTION
**Pull request by rddheath**

### CODE REVIEW

File: modules/ROOT/pages/import-mule-app-project.adoc [hunk 1]
```diff
@@ -12,6 +12,15 @@ You can import a Mule-application project that was exported from Design Center.
 * Importing a project that was exported from a different environment within a single organization
 * Importing a project that was exported from an environment that is in a different organization
 
+[IMPORTANT]
+====
+An exported project does not include items that the project linked to in its original location.
```
Comment (dnp1988) [inline modules/ROOT/pages/import-mule-app-project.adoc]: Actually when exporting a project, the pom inside the jar has the references to its external dependencies. When importing the project, the external dependencies in the pom will me omitted.

File: modules/ROOT/pages/import-mule-app-project.adoc [hunk 2]
```diff
@@ -12,6 +12,15 @@ You can import a Mule-application project that was exported from Design Center.
...
+
+For example, if the project contains a Database connector, the database driver for that connector is not included in the exported project. After you import the project, you must open the Database connector, browse to the location of the database driver, and select it.
```
Comment (dnp1988) [inline modules/ROOT/pages/import-mule-app-project.adoc]: Again, the driver is referenced in the pom of the exported project. It will be omitted during import and the user will have to set it manually. It is possible that the user might not have a suitable driver in his organization. In that case he will need to upload a new one.

File: modules/ROOT/pages/import-mule-app-project.adoc [hunk 3]
```diff
@@ -12,6 +12,15 @@ You can import a Mule-application project that was exported from Design Center.
...
+
+If the project contains any connectors that require a driver or another type of library, you must open those connectors and select the required libraries.
```
Comment (dnp1988) [inline modules/ROOT/pages/import-mule-app-project.adoc]: I would rephrase this to 'you must open those connector configurations and set up the required libraries' or something similar. The thing I want to emphasize is that drivers and libraries are normally set up in the connector reusable configurations.

File: modules/ROOT/pages/import-mule-app-project.adoc (hunk 1 above)
Comment (rddheath) [reply modules/ROOT/pages/import-mule-app-project.adoc]: I changed "exported" to "imported". I've also made a number of other related changes.

**Comment by badiazad** [comment]
This looks good to me now - after you add Salsa's suggestions
